pkg/dockerutil/dockerutils.go

```diff
@@ -712,6 +713,10 @@ func ComposeCmd(cmd *ComposeCmdOpts) (string, string, error) {
 		arg = append(arg, "-f", file)
 	}
 
+	if cmd.Profiles != nil {
+		arg = append(arg, "--profile", strings.Join(cmd.Profiles, ","))
+	}
```

Comment:
I think it won't work like this (although the code works, but only with one profile).

This should work:

```suggestion
	for _, profile := range cmd.Profiles {
		arg = append(arg, "--profile", profile)
	}
```

Reply:
Interesting, I wasn't actually thinking of offering more than one profile. Probably shouldn't have been cmd.Profiles. Do you think there's a need for more than one, since a single profile can easily start multiple services?

Reply:
I think having more than one profile is fine, especially since we're not enforcing a specific profile name.

People are unlikely to name their profiles the same across different add-ons. So, if someone pulls multiple add-ons with profiles and starts them all, the profiles will likely differ. Instead of running `ddev start --profile` for each one, they could start them all with a single command.

Reply:
`ddev start --profile="*"` is useful, but having something more specific is even better. For example, starting only certain profiles, like this `ddev start --profile=xhgui,extra` allows you to start some profiles while excluding others.